handbook/company/README.md

```diff
@@ -15,6 +15,14 @@ We are dedicated to
 
 ## Culture
 
+### Scheduling with CEO
```

Comment:
Can this go on the sub-page we discussed w/ the other EA-focused stuff?

This "Culture" section is at the top of the page and is viewed by candidates, etc.  It is more enticing to highlight all-remote, open source, and our values than scheduling logistics.

Reply:
Removing from this page and adding to https://fleetdm.com/handbook/people/ceo-handbook